site/en/docs/web-platform/view-transitions/index.md

```diff
@@ -1001,6 +997,10 @@ You can also provide some `classNames` to add to `<html>` during the transition,
 
 You can also pass `true` to `skipTransition` if you don't want an animation, even in browsers that support View Transitions. This is useful if your site has a user preference to disable transitions.
 
+## React/Preact hook
+
+React doesn't provide an easy way to be notified when a DOM update is complete, but the complexity can be [wrapped up in a hook](https://gist.github.com/jakearchibald/8437ee8247177ab0d8f171fecba0a628).
```

Comment:
Not sure what the best practice is for this. It's too big to go in the article itself, but it's something partners have asked for.